app/presenters/unauthorized_presenter.rb

```diff
@@ -1,30 +1,34 @@
 # frozen_string_literal: true
 class UnauthorizedPresenter
   def initialize(id)
-    @asset = GenericWork.find(id)
+    @resource = GenericWork.find(id)
```

Comment:
Consider creating a SolrDocument here and then you could use methods for all the terms below.
`@resource = SolrDocument.new(GenericWork.find(id).to_solr)`
then delegate all your methods to resource:
`delegate :title, :family_name, :given_name, :thumbnail, to: :resource`
see https://github.com/aic-collections/aicdams-lakeshore/blob/master/app/models/concerns/solr_document_extensions.rb#L68-L92